bower.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "vaadin-upload",
+  "homepage": "https://vaadin.com/elements",
+  "authors": [
+    "Vaadin Ltd"
+  ],
+  "description": "Polymer Element for uploading files",
+  "main": "vaadin-upload.html",
+  "moduleType": [
+    "globals"
+  ],
+  "keywords": [
+    "Vaadin",
+    "upload",
+    "web-components",
+    "web-component",
+    "polymer"
+  ],
+  "license": "Apache-2.0",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests",
+    "demo"
+  ],
+  "devDependencies": {
+    "web-component-tester": "^4.1.0",
```

Comment:
Do we need 4.1.0 or is it enough ^4.0.0? the same with every dependency below I would use ^1.0.0 that is what polymer elements have